.github/workflows/build.yml

```diff
@@ -56,6 +56,13 @@ jobs:
           sudo apt-get install -y autoconf automake pkg-config g++-mingw-w64 gcc-multilib python3-pip ninja-build nasm wine wine32 wine64 ccache
           sudo python3 -m pip install meson
 
+      - name: Install llvm-mingw
```

Comment:
I think it is fine to use llvm-mingw, but this should be another build, we should still test gcc mingw for compatibility reasons. Even though llvm toolchain is more suited for windows cross compilation these days, I'm sure there are people who wants to just use gcc mingw provided in their distro.

You can add `toolchain: [gcc, llvm]` as matrix dimension I think.

Reply:
The rule of thumb is that we support mingw version packaged in the latest Ubuntu LTS.